src/components/ContactCard/ContactForm/ContactFormField.jsx

```diff
@@ -82,7 +82,7 @@ class ContactFormField extends React.Component {
             onClick={() => this.addField(fields)}
             className="contact-form__meta-button contact-form__meta-button--add"
           >
-            <Icon icon={'plus'} color="white" width="10" height="10" />
+            <Icon icon={'plus-small'} color="white" width="10" height="10" />
```

Comment:
For the plus icon above, we changed the size from 10 to 12, but not here. Is it normal?

Reply:
Yep, it looks great that way ;).

Reply:
Best reason 😎